sdk/src/pubkey.rs

```diff
@@ -130,7 +124,7 @@ impl Pubkey {
         // not supported
         #[cfg(not(all(feature = "program", target_arch = "bpf")))]
         {
-            let mut hasher = Hasher::default();
+            let mut hasher = crate::hash::Hasher::default();
```

Comment:
Na, my goal here was to remove all those feature-specific `use` statements at the top of pubkey.rs, to clear the way for easier future refactoring at #12987 and #12989